<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QSplitter" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="orientation">
   <enum>Qt::Horizontal</enum>
  </property>
  <property name="childrenCollapsible">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="title">
    <string>View</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
    <item>
     <widget class="QSlider" name="slider_zoom">
      <property name="minimum">
       <number>-2</number>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="sliderPosition">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Endpoints</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>L1 P1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l2x2" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>L2 P1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l2x1" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>L1 P2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>L2 P2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l1x2" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l1x1" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l1y1" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l1y2" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l2y2" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="FlexibleDoubleSpinBox" name="dsb_l2y1" native="true">
         <property name="decimals" stdset="0">
          <number>20</number>
         </property>
         <property name="minimum" stdset="0">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum" stdset="0">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Preset</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="cb_presets">
         <item>
          <property name="text">
           <string>&lt;Manual&gt;</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X-Pt&lt;/p&gt;&lt;p&gt;(V2)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="ResultWidget" name="result_flsiV2" native="true"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="ResultWidget" name="result_flsiOrig" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X-Pt&lt;/p&gt;&lt;p&gt;(Orig.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="ResultWidget" name="result_flsiTweaked" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X-Pt&lt;/p&gt;&lt;p&gt;(Orig. Tweaked)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="2">
        <widget class="ResultWidget" name="result_gaussElim" native="true"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X-Pt&lt;/p&gt;&lt;p&gt;(Gauss. Elim.)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="1" colspan="2">
        <widget class="ResultWidget" name="result_eddy" native="true"/>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X-Pt&lt;/p&gt;&lt;p&gt;(Eddy)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlexibleDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>gui/flexibledoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ResultWidget</class>
   <extends>QWidget</extends>
   <header>gui/widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
